tests/test_code.py

```diff
@@ -36,9 +36,12 @@ def _get_lintable_files(self, folder):
     def test_linting(self):
         files = self._get_lintable_files('friskby_controlpanel')
         files = files + self._get_lintable_files('tests')
+        files = files + self._get_lintable_files('bin')
         for f in files:
             self.assertEqual(lint.lint(f), 0, 'linting required for %s' % f)
 
 
+            
```

Comment:
there's suddenly two new lines here, one with trailing whitespace.  Please run `M-x whitespace-cleanup` on this file, or use git tools to remove.